apps/cowswap-frontend/src/legacy/hooks/useContract.ts

```diff
@@ -111,7 +115,12 @@ export function useEIP2612Contract(tokenAddress?: string): Contract | null {
 }
 
 export function useInterfaceMulticall() {
-  return useContract<UniswapInterfaceMulticall>(MULTICALL_ADDRESS, MulticallABI, false) as UniswapInterfaceMulticall
+  return useContract<UniswapInterfaceMulticall>(
+    MULTICALL_ADDRESS,
+    MulticallABI,
+    false,
+    true
```

Comment:
Now we always use `networkConnection` for multicalls

Reply:
But why, to make it more reliable? 
I guess we might see some increse in Infura usage?

Reply:
Reverted this, can't estimate influence of these changes on Infura load